LaunchAtLogin/copy-helper.sh

```diff
@@ -18,5 +26,5 @@ fi
 
 if [[ $CONFIGURATION == "Release" ]]; then
 	rm -rf "$origin_helper_path"
```

Comment:
I think an `|| true` also needs to be added here. Basically what happens is that on first "Release" or "Archive" build the script deletes the files, any consecutive build thinks that the Frameworks has already been copied over and skips copying the framework again, hence the files are already missing from there...

Reply:
If that folder isn't present, there'd be a message printed to `stderr` but it shouldn't cause the script to fail. (Since it's not the last command in the script, it doesn't determine the script's exit code.) Are you seeing the script fail bc of this?